app/posts/publish-teacher-training-courses/2021-06-03-placement-schools-and-training-centres.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Placement schools and training centres
+description: Asking providers to give more detailed location information
+date: 2021-06-14
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+## The location problem
+
+Currently, Publish uses a model inherited from UCAS. This allows courses to show 2 types of location:
+
+* the training provider’s contact address
+their training locations – places a candidate
+* can apply to, but not necessarily where they will spend the majority, if any, of their time.
```

Comment:
Looks like this bullet point is in the wrong place. Should it be in front of 'their training locations...'?